docs/reference/ml/apis/update-datafeed.asciidoc

```diff
@@ -86,7 +86,7 @@ The following properties can be updated after the {dfeed} is created:
   The <<ml-detectorconfig,detector configuration objects>> in a job can contain
   functions that use these script fields.
   For more information,
-  see {ref}/search-request-script-fields.html[Script Fields].
+  see {ref}/search-request-body.html#search-request-script-fields[Script Fields].
```

Comment:
Not sure why these were external links in the first place, so I'm suggesting we convert them to internal links:
```suggestion
  see <<search-request-script-fields,Script fields>>.
```